data/Makefile.am

```diff
@@ -75,6 +74,10 @@ filetypes = \
 	filedefs/filetypes.yaml \
 	filedefs/filetypes.zephir
 
+# generated filetypes
+filetypes_nodist = \
+	filedefs/filetypes.python
```

Comment:
I think `filetypes.python` should be added to `.gitignore` or such now.

Reply:
Done in 6ffaf515.